cognite/client/data_classes/sequences.py

```diff
@@ -276,21 +276,42 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
             dumped = {utils._auxiliary.to_camel_case(key): value for key, value in dumped.items()}
         return {key: value for key, value in dumped.items() if value is not None}
 
-    def to_pandas(self) -> "pandas.DataFrame":
+    def to_pandas(self, column_names: str = "columnExternalId") -> "pandas.DataFrame":
         """Convert the sequence data into a pandas DataFrame.
 
         Args:
-            column_names (str): Which field to use as column header. Either "externalId", "id" or "externalIdIfExists" for externalId if it exists for all columns and id otherwise.
+            column_names (str):  Which field(s) to use as column header. Can use "externalId", "id", "columnExternalId", "id|columnExternalId" or "externalId|columnExternalId".
 
         Returns:
             pandas.DataFrame: The dataframe.
         """
         pd = utils._auxiliary.local_import("pandas")
 
+        if column_names not in [
+            "externalId",
+            "id",
+            "columnExternalId",
+            "id|columnExternalId",
+            "externalId|columnExternalId",
+        ]:
+            raise ValueError(
+                'Invalid column_names value, should be one of "externalId", "id", "columnExternalId", "id|columnExternalId", "externalId|columnExternalId"'
+            )
```

Comment:
```
options = [
            "externalId",
            "id",
            "columnExternalId",
            "id|columnExternalId",
            "externalId|columnExternalId",
        ]
if column_names not in options:
    raise ValueError(f"Invalid column_names value, should be one of {options}")
```

Reply:
fixed in a py3.5 compatible way (although using f-strings would be awesome)